api/envoy/service/ratelimit/v2/rls.proto

```diff
@@ -103,4 +103,7 @@ message RateLimitResponse {
 
   // A list of headers to add to the response
   repeated api.v2.core.HeaderValue headers = 3;
+
+  // A list of headers to add to the upstream server
```

Comment:
This sentence and field name don't clearly convey the intention here I think. We're not adding headers to a server.. do you mean to say something like "A list of headers to add to the request when forwarding" and name it `request_headers_to_add`? Also, the existing `headers` field should have a `[#next-major-version: rename to response_headers]` or the like added.

Reply:
Thanks! Have renamed to `request_headers_to_add` with a comment for existing `headers` field.